src/Job/StaticCacheFullBuildJob.php

```diff
@@ -61,7 +64,8 @@ public function process()
                 break;
             }
             $meta = Publisher::singleton()->publishURL($url, true);
-            if (!empty($meta['success'])) {
+            // Includes ErrorPage where you can choose your own status codes
+            if (!empty($meta['success']) || $meta['responsecode'] > 400) {
```

Comment:
This seems problematic - surely if we get a >400 code response we assume something has gone wrong? Is there a way to check explicitly that the response is coming from an errorpage?

Reply:
Good point, I have to dig deeper from this point. With this extra logic, you will end up in a loop thus exhausting memory and mark as broken job.

Reply:
Is there a way to check that page type here to see if it's an ErrorPage instead?

Reply:
Reverting, this fix is not needed anymore.